src/pages/waiting-room-to-car/components/eyesight-failure-confirmation/eyesight-failure-confirmation.scss

```diff
@@ -0,0 +1,25 @@
+eyesight-failure-confirmation {
+  width: 100%;
+  #eyesight-failure-confirmation {
+    margin: 12px;
+    padding: 40px 55px;
+    background: map-get($colors-gds, 'gds-blue');
+    color: map-get($colors, 'mes-white');
+    #eyesight-failure-icon {
+      height: 68px;
+      margin-right: 15px;
+    }
+    #eyesight-failure-title {
+      color: inherit;
+    }
+    #eyesight-failure-subtitle {
+      margin-top: 18px;
+      color: inherit;
+    }
+    button {
+      width: 230px;
+      height: 56px;
```

Comment:
You raise a good point, perhaps it would be better to size buttons with `em`s for this exact reason? Either way, I think we certainly need to assign a size in this instance to ensure that both buttons are the same size despite them having different amounts of text content. Let's raise it as a discussion with the rest of the team.